src/utils/errors.ts

```diff
@@ -110,7 +108,7 @@ export class AccessTokenError extends AuthError {
 /**
  * @ignore
  */
-export type HandlerErrorCause = Error | AuthError | HttpError;
+export type HandlerErrorCause = Error | AuthError | (Error & { status: number; statusCode: number });
```

Comment:
Should `Error & { status: number; statusCode: number }` be its own type?

Reply:
Sure - I'll make an `HttpError` type